pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSchemas.java

```diff
@@ -78,10 +78,14 @@ void run() throws Exception {
         @Parameter(description = "persistent://tenant/namespace/topic", required = true)
         private java.util.List<String> params;
 
+        @Parameter(names = { "-f",
```

Comment:
From a user's perspective, when should they delete schema forcefully?
Can we delete all schema forcefully?

Reply:
> From a user's perspective, when should they delete schema forcefully?

A case I have met is all replicas of the ledger corresponding to the schema of the health-check topic were deleted (during the offline process of bookies), resulting in the failure of health-check and the failure is unrecoverable(can't update or delete the schema). I think the common topics also have the same problem.

> Can we delete all schema forcefully?

Do you mean all schemas for all topics? No, we can't

Reply:
> > Can we delete all schema forcefully?
> 
> Do you mean all schemas for all topics? No, we can't

I mean, is it safe to always use "delete -f" instead of normal delete?  Or delete forcefully should only be used to recover unexpected state (like your case, schema data deleted by offline process).

Reply:
> I mean, is it safe to always use "delete -f" instead of normal delete? 
It's not safe here.
If we delete a schema like before, there will leave a new ledger for the schema. But with `-f`, all ledgers will be removed. 
One place the ledgers of the schema shown is `internal-stats`,  with or without `-f`, we will get a different result.
> Or delete forcefully should only be used to recover unexpected state (like your case, schema data deleted by offline process).
Delete forcefully should be used if we want to delete schema completely or to recover an unexpected state